product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,225 @@
+
+### Validate the migration
+
+Finally, you can follow the progression of the migration. First, use `\dt+;` while logged into the older BigAnimal Postgres instance to see how much data is to be replicated. In this example:
+
+```
+                                           List of relations
+ Schema |       Name       | Type  |   Owner   | Persistence | Access method |    Size    | Description 
+--------+------------------+-------+-----------+-------------+---------------+------------+-------------
+ public | pgbench_accounts | table | edb_admin | permanent   | heap          | 1572 MB    | 
+ public | pgbench_branches | table | edb_admin | permanent   | heap          | 8192 bytes | 
+ public | pgbench_history  | table | edb_admin | permanent   | heap          | 0 bytes    | 
+ public | pgbench_tellers  | table | edb_admin | permanent   | heap          | 120 kB     | 
+```
+
+Then run `\dt+;` while logged into the new BigAnimal instance and compare:
+
+```
+                                         List of relations
+ Schema |       Name       | Type  |   Owner   | Persistence | Access method |  Size   | Description 
+--------+------------------+-------+-----------+-------------+---------------+---------+-------------
+ public | pgbench_accounts | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_branches | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_history  | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_tellers  | table | edb_admin | permanent   | heap          | 0 bytes | 
+```
+
+If logical replication is running correctly, each time you run `\dt+;` you see that more data has been migrated:
+
+```
+                                         List of relations
+ Schema |       Name       | Type  |   Owner   | Persistence | Access method |  Size   | Description 
+--------+------------------+-------+-----------+-------------+---------------+---------+-------------
+ public | pgbench_accounts | table | edb_admin | permanent   | heap          | 344 MB  | 
+ public | pgbench_branches | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_history  | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_tellers  | table | edb_admin | permanent   | heap          | 0 bytes | 
+```
+
+#### LiveCompare
+
+A final validation step to ensure the two databases are identical is to use [EDB's LiveCompare](https://www.enterprisedb.com/docs/livecompare/latest/). 
+
+LiveCompare compares the databases, generates a comparison report, and provides data manipulation language (DML) scripts so you can optionally apply the DML and fix any database inconsistencies.
```

Comment:
Add a space between the previous section and a Note for callout.  

To:  You can optionally use (LC link) to generate a comparison report of the source and target database to validate that all database objects and data are consistent.

Reply:
👍

Reply:
Recommend adding this LC section into a Note box to place emphasis on an EDB product in an otherwise non-EDB process.